Example/RoboKittenTests/Templates/AutoRandom.stencil

```diff
@@ -0,0 +1,51 @@
+import Foundation
+@testable import Betterme
```

Comment:
same

Reply:
unfortunately I didn't found solution for that yet

Reply:
You can pass parameter to sourcery script

Reply:
ah, just wanted to answer the same :)

there's `args` that you can pass and read inside

or better - setup yml file with all configs and pass path to it via `--config` flag

so that it'll contain

```yml
args:
  testable:
    - Betterme
```